diff --git test/Makefile.in test/Makefile.in
index 93c0d5b..6d11143 100644
--- test/Makefile.in
+++ test/Makefile.in
@@ -1346,7 +1346,7 @@ check_SCRIPTS = $(TEST_SCRIPT)
 # executed, generally most specific tests to least specific tests.
 # As an exception, long-running tests should occur earlier in the list.
 # This gives them more time to run when tests are executing in parallel.
-TEST_PROG = testhdf5 cache_api cache_tagging lheap ohdr stab gheap \
+TEST_PROG = testhdf5 cache cache_api cache_tagging lheap ohdr stab gheap \
            farray earray btree2 fheap \
            pool accum hyperslab istore bittests dt_arith \
            dtypes dsets cmpd_dset filter_fail extend external efc objcopy links unlink \
